pyrakoon/compat/client/functional.py

```diff
@@ -13,7 +13,7 @@
 from .socket import ArakoonSocketClient
 from ..consistency import Consistency, Consistent, NoGuarantee, AtLeast
 from ..utils import convert_exceptions as _convert_exceptions, validate_signature as _validate_signature
-from ..sequence import Set, Delete, DeletePrefix, Assert, AssertExists, Replace, Sequence
+from ..sequence import Set, Delete, DeletePrefix, Assert, AssertExists, Replace, Sequence, AssertRange
```

Comment:
Missing a non-sequence assert method in the ArakoonClient
`def assertPrefix(prefix, keys)`

Reply:
can you provide a use case for this? 
(I think it will always boils down to adding an `AssertPrefix` to a sequence)